05week/spaceTravelToMars.js

```diff
@@ -10,6 +10,85 @@ let jobTypes = {
 };
 
 // Your code here
+class CrewMember {
+  constructor(name, job, specialSkill, ship){
+    this.name = name;
+    this.job = job;
+    this.specialSkill = specialSkill;
+    this.ship = null;
+  }
+  enterShip(ship) {
+    if(this.ship == null){
+    this.ship = ship;
+    ship.crew.push(this)
+    console.log(`${this.name} is starting on the ${this.ship.shipName}`)
+    // console.log(ship.crew)
+    // console.log(this)
+    } else  {
+      // console.log(ship.crew)
+      ship.crew.pop()
```

Comment:
what if the this crew member was not at the end of that ship? You should remove the correct crew memeber, not just the last one